### PR TITLE
Ready to Merge for zh translation

### PR DESCRIPTION
**Pull request by RichardStark**

### CODE REVIEW

**Comment by RichardStark** [comment]
new fix finished @ 2020.04.11

**Comment by RichardStark** [comment]
new merge & fix finished @ 2020.04.18，Merged&Fixed all the zh content according to the en content since the last merge

**Comment by RichardStark** [comment]
modified the editlink for zh version

**Comment by RichardStark** [comment]
modified zh content according to the en merge

**Comment by marklaatikainen** [comment]
in _layout.js_ and _404.js_ indexOf should be compared to _> -1_ instead of 0

File: src/components/layout.js [hunk 1]
```diff
@@ -13,7 +13,11 @@ class Layout extends Component {
 
   componentDidMount() {
     const siteLanguage =
-      window.location.pathname.indexOf('/en') !== -1 ? 'en' : 'fi';
+      window.location.pathname.indexOf('/en') > 0
```
Comment (marklaatikainen) [inline src/components/layout.js:16]: IndexOf should be compared with -1 instead of 0

File: src/components/layout.js [hunk 2]
```diff
@@ -13,7 +13,11 @@ class Layout extends Component {
...
+        ? 'en'
+        : window.location.pathname.indexOf('/zh') > 0
```
Comment (marklaatikainen) [inline src/components/layout.js:18]: IndexOf should be compared with -1 instead of 0

File: src/pages/404.js [hunk 3]
```diff
@@ -14,7 +14,11 @@ class NotFoundPage extends Component {
 
   componentDidMount() {
     const siteLanguage =
-      window.location.pathname.indexOf('/en') !== -1 ? 'en' : 'fi';
+      window.location.pathname.indexOf('/en') > 0
```
Comment (marklaatikainen) [inline src/pages/404.js:17]: IndexOf should be compared with -1 instead of 0

File: src/pages/404.js [hunk 4]
```diff
@@ -14,7 +14,11 @@ class NotFoundPage extends Component {
...
+        ? 'en'
+        : window.location.pathname.indexOf('/zh') > 0
```
Comment (marklaatikainen) [inline src/pages/404.js:19]: IndexOf should be compared with -1 instead of 0

**Comment by RichardStark** [comment]
Fixed bugs, Thx @marklaatikainen

**Comment by mluukkai** [comment]
merged

**Pull request closed without merging**